src/ngx_rewrite_driver_factory.cc

```diff
@@ -151,6 +151,10 @@ RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptions() {
   return options;
 }
 
+RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptionsForQuery() {
+  return new NgxRewriteOptions(thread_system());
```

Comment:
why do we even have the one that sets Core Filters?  We don't have one of those in apache_config.cc.